manim/utils/space_ops.py

```diff
@@ -221,3 +221,19 @@ def get_winding_number(points):
         d_angle = ((d_angle + PI) % TAU) - PI
         total_angle += d_angle
     return total_angle / TAU
+
+
+def shoelace(x_y, absoluteValue=False, orientation=True):
```

Comment:
This should be split into two functions. Boolean parameters are generally bad and python variables should be in snake_case.

Reply:
The snake case is obvious.
I suppose I can split this into only a function returning the non absolute value and one that uses this function to return the orientation? Seems better indeed but I'll sleep over how to do that exactly.